src/Components/App/App.css

```diff
@@ -3,7 +3,7 @@ body {
   padding: 0;
   /* overflow: hidden; */
   position: relative;
-  background: url('/src/images/pexels-pixabay-53154.jpg') no-repeat center center fixed;
+  background: url('/src/Components/images/pexels-pixabay-53154.jpg') no-repeat center center fixed;
```

Comment:
I think we will end up moving the images folder out of the Components after we connect, which will be a simple refactor.

Reply:
agreed!

Reply:
I agree since it is not a component.